src/alpaka/AlpakaDataFormats/SiPixelDigisAlpaka.h

```diff
@@ -10,14 +10,14 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
   class SiPixelDigisAlpaka {
   public:
     SiPixelDigisAlpaka() = default;
-    explicit SiPixelDigisAlpaka(Device const &device, size_t maxFedWords)
-        : xx_d{::cms::alpakatools::allocDeviceBuf<uint16_t>(device, maxFedWords)},
-          yy_d{::cms::alpakatools::allocDeviceBuf<uint16_t>(device, maxFedWords)},
-          adc_d{::cms::alpakatools::allocDeviceBuf<uint16_t>(device, maxFedWords)},
-          moduleInd_d{::cms::alpakatools::allocDeviceBuf<uint16_t>(device, maxFedWords)},
-          clus_d{::cms::alpakatools::allocDeviceBuf<int32_t>(device, maxFedWords)},
-          pdigi_d{::cms::alpakatools::allocDeviceBuf<uint32_t>(device, maxFedWords)},
-          rawIdArr_d{::cms::alpakatools::allocDeviceBuf<uint32_t>(device, maxFedWords)} {}
+    explicit SiPixelDigisAlpaka(Queue queue, size_t maxFedWords)
```

Comment:
I see `Queue` is taken by reference elsewhere, is copy here intentional?

Reply:
No... thanks for catching it. I've tested that passing by reference works fine (here and in two more places).